string subscription
string bond_id
float64 frequency
